deps/github.com/arangodb/arangosync/client/api.go

```diff
@@ -0,0 +1,333 @@
+//
+// Copyright 2017 ArangoDB GmbH, Cologne, Germany
+//
+// The Programs (which include both the software and documentation) contain
+// proprietary information of ArangoDB GmbH; they are provided under a license
+// agreement containing restrictions on use and disclosure and are also
+// protected by copyright, patent and other intellectual and industrial
+// property laws. Reverse engineering, disassembly or decompilation of the
+// Programs, except to the extent required to obtain interoperability with
+// other independently created software or as specified by law, is prohibited.
```

Comment:
This looks different than copyright and Apache licenses in C++ sources.  Is that proper?